test/apiv2/test-apiv2

```diff
@@ -380,13 +380,26 @@ function start_service() {
         die "Cannot start service on non-localhost ($HOST)"
     fi
 
+    # FIXME: EXPERIMENTAL: 2022-06-13: podman rootless needs a namespace. If
+    # system-service is the first podman command run (as is the case in CI)
+    # this will happen as a fork-exec, where the parent podman creates the
+    # namespace and the child is the server. Then, when stop_service() kills
+    # the parent, the child (server) happily stays alive and ruins subsequent
+    # tests that try to restart service with different settings.
+    # Workaround: run an unshare to get namespaces initialized.
+    if [[ $(id -u) != 0 ]]; then
+        # Complicated parens & redirection are to prevent job-control clutter
+        ( $PODMAN_BIN unshare sleep 10 & ) &>/dev/null
+    fi
+
```

Comment:
@giuseppe @nalind with this addition, CI is now passing. At your convenience, PTAL and see if this is the best (or least-worst) solution, and if the comment is accurate. Thank you for your help in this!

Reply:
I think `podman unshare true` should be enough, the rootless pause process pid is never killed.

Reply:
So done, and CI is green. Thanks for the suggestion, that is much cleaner.